package.json

```diff
@@ -6,7 +6,8 @@
     "start": "expo start",
     "android": "expo start --android",
     "ios": "expo start --ios",
-    "web": "expo start --web"
+    "web": "expo start --web",
+    "format:prettier": "prettier --write . --ignore-path .gitignore"
```

Comment:
note: Regarding the `--ignore-path` as this project is small, it's okay. In bigger monorepos it would be trickier to manage

Reply:
True, I might have to change it later on. It will come down to how we structure the rest of the project